bindings/python/linkgrammar.py

```diff
@@ -29,7 +29,13 @@ def __init__(self, verbosity=0,
                  spell_guess=False,
                  use_sat=False,
                  max_parse_time=-1,
-                 disjunct_cost=2.7):
+                 disjunct_cost=2.7,
+            #     debug="",
+            #     test="",
+                 use_viterbi=False,
+                 use_cluster_disjuncts=False,
```

Comment:
The viterbi code does not work, and is unlikely to, in the next few years, if ever.  Thus, it is not suitable for the API.

The cluster-disjuncts code works, but the database for is is 7 years out-of-date, and it is unlikely that it will ever be rebuilt.  Thus, this is not suitable for the API.